src/main/java/com/example/commercetoolEx/controller/CategoryController.java

```diff
@@ -0,0 +1,42 @@
+package com.example.commercetoolEx.controller;
+
+import com.commercetools.api.models.category.Category;
+import com.commercetools.api.models.category.CategoryPagedQueryResponse;
+import com.example.commercetoolEx.dto.category.AddCategoryRequest;
+import com.example.commercetoolEx.dto.category.UpdateCategoryRequest;
+import com.example.commercetoolEx.service.CategoryService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+import reactor.core.publisher.Mono;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping(path = "/category")
```

Comment:
the REST API resource mapping should be in plural. `categories` instead of `category`